packages/core/helper-plugin/src/components/Status.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { StatusVariant, ThemeColors as DesignSystemThemeColors } from '@strapi/design-system';
```

Comment:
`DesignSystemThemeColors` not to be confused with `ThemeColors` from `styled-components`

Reply:
I could not find Theme colors exported from DS, We have created [issue](https://strapi-inc.atlassian.net/browse/CS-187) regarding the same.

Reply:
@madhurisandbhor it's not available from https://github.com/strapi/design-system/blob/main/packages/strapi-design-system/styled.d.ts#L42?

Reply:
> I could not find Theme colors exported from DS, We have created [issue](https://strapi-inc.atlassian.net/browse/CS-187) regarding the same.

@Feranchz @markkaylor Could you please check this issue, thanks.

Reply:
Good catch. I'll look into it. I think we should just actually export the types before merging this, right?

Reply:
Yes It should work then.

Reply:
The DS types have been updated and looks like this is working now.